src/i18n/en.js

```diff
@@ -53,19 +53,19 @@ export default {
         You’ll find an Email with steps to follow.
       </p>
       `,
-    description: `Do not worry! It happens to everyone.`,
-    description2: `Enter your email and we will help you`,
+    description: `Don’t worry! It happens.`,
+    description2: `Enter your Email and we’ll be glad to help you.`,
```

Comment:
@carodipietro could you confirm what is the desired apostrophe? We need to unify it in all the texts.